src/Momento.Sdk/Exceptions/BadRequestException.cs

```diff
@@ -1,14 +1,14 @@
 ﻿namespace Momento.Sdk.Exceptions;
 
+using System;
+
 /// <summary>
 /// Invalid parameters sent to Momento Services.
 /// </summary>
 public class BadRequestException : SdkException
 {
-    public BadRequestException(string message) : base(MomentoErrorCode.BAD_REQUEST_ERROR, message)
-    {
-    }
-    public BadRequestException(string message, MomentoErrorTransportDetails transportDetails) : base(MomentoErrorCode.BAD_REQUEST_ERROR, message, transportDetails)
+    public BadRequestException(string message, MomentoErrorTransportDetails? transportDetails=null, Exception? e=null) : base(MomentoErrorCode.BAD_REQUEST_ERROR, message, transportDetails, e)
     {
+        this.MessageWrapper = "The request was invalid; please contact Momento";
```

Comment:
for this one in particular, things may have changed a bit since the time i wrote the spec.  For this one we are definitely expecting the gRPC error from the server to contain more information about why the request was invalid, so we want to try to make sure we surface that as part of the most visible user-facing string.  So perhaps we want to interpolate it into this string?